etna/packages/etna-js/upload/api/__tests__/upload_api.spec.jsx

```diff
@@ -3,10 +3,6 @@ import * as api from '../upload_api';
 import { mockStore, stubUrl, cleanStubs } from '../../../spec/helpers';
 
 describe('upload api', () => {
-  afterEach(() => {
-    cleanStubs();
-  });
-
   it('postAuthorizeUpload makes a request to the authorize upload route', () => {
     global.fetch = require('node-fetch');
```

Comment:
nit: Should be able to get rid of this `fetch` assignment, since you now assign it in `setup.js`?

Reply:
Yup

Reply:
Taken care of.